### PR TITLE
Build issue with cabal 1.22+ on Mac OS X

### PR DESCRIPTION
**Pull request by cchantep**

### CODE REVIEW

File: Graphics/UI/Gtk/Poppler/Structs.hsc [hunk 1]
```diff
@@ -21,6 +21,7 @@
 --
 -- #hide
 
+#include <stdio.h>
```
Comment (cchantep) [inline Graphics/UI/Gtk/Poppler/Structs.hsc]: Just to remove warnings about `printf`

**Comment by cchantep** [comment]
See also headers updated on OS X 10.10 to be able to build this hackage: https://gist.github.com/cchantep/d4fc2a766fb5b93042b4

**Comment by wavewave** [comment]
Cabal issue has been dealt with #9 in the same way.

**Comment by wavewave** [comment]
Could you provide what errors you had with OS X 10.10 if you do not patch the system header files? If you can show us diff between the original and the new ones, that's very helpful to me. Thanks!

**Comment by FranklinChen** [comment]
@cchantep Any news on this?

**Comment by wavewave** [comment]
close this.

**Pull request closed without merging**